src/styles/globals.css

```diff
@@ -292,6 +292,14 @@
          border-image-slice: 1 !important;
          border-image-source: linear-gradient(90.07deg, #5033e1 3.77%, #a112ff 42.1%, #503cdc 94.28%);
      }
+
+     .sg-border-gradient-hubspot {
```

Comment:
The border gradient should have 40% opacity. Can we give it a different name like `.sg-border-gradient-light` ?

Reply:
Hi @traceyljohnson, we will set the opacity to 40% and rename the class to .sg-border-gradient-light.
Thank you